src/components/pages/Data.tsx

```diff
@@ -1,164 +1,152 @@
-            </a>
-            .
-          </p>
+        <div
+          className={
+            isMobileDeviceOrTablet ? "static-mobile" : "static-content"
+          }
+        >
           <h1>{Translate("OurData")}</h1>
-          <p>{Translate("OurDataText", ["Text"])}</p>
-          <p>
-            {Translate(
-              "DataDictionaryContent",
-              ["DataDictionaryContentParagraphOne"],
-              null,
-              [false, true]
-            )}
-            <a
-              rel="noopener noreferrer"
-              onClick={() => clickLink("Manuscript Appendix")}
-              target="_blank"
-              href="https://drive.google.com/file/d/1d8-U0NgjVBTDzdj3rbAYfZyYVp0HRhgj/view?usp=sharing"
-            >
-              {Translate(
-                "ManuscriptAppendix",
-                ["ManuscriptAppendixParagraphOne"],
-                null,
-                [false, true]
-              )}
-            </a>
-            {Translate(
-              "ManuscriptAppendix",
-              ["ManuscriptAppendixParagraphTwo"],
-              null,
-              [false, true]
-            )}
-            {Translate(
-              "DataDictionaryContent",
-              ["DataDictionaryContentParagraphTwo"],
-              null,
-              [false, true]
-            )}
-            <a
-              rel="noopener noreferrer"
-              onClick={() => clickLink("Data Dictionary")}
-              target="_blank"
-              href="https://docs.google.com/spreadsheets/d/1KQbp5T9Cq_HnNpmBTWY1iKs6Etu1-qJcnhdJ5eyw7N8/edit?usp=sharing"
-            >
-              {Translate("DataDictionary")}
-            </a>
-            .
-          </p>
-          <p>
-            {Translate("DataDownload", null, null, [false, true])}
-            <Link to="/About">{Translate("SeroTrackerTeam")}</Link>.
-          </p>
+          <p>{Translate("WhatWeDoText", ["FirstParagraph"])}</p>
+          <p>{Translate("WhatWeDoText", ["SecondParagraph"])}</p>
+          <br></br>
+          <DataButtons />
+          <br></br>
+          <DataDropdowns />
+          <br></br>
+          <h1 className="mb-0">{Translate("ReferencesTable")}</h1>
         </div>
-
         <div
           className={
-            isMobileDeviceOrTablet ? "pb-3 pt-3" : "pb-3 pt-3 reference-table"
+            isMobileDeviceOrTablet
+              ? "pb-3 pt-3 static-mobile"
+              : "pb-3 pt-3 static-content"
           }
         >
           <Segment>
-            <Loader indeterminate active={isLoading} />
-            <StudiesTable
-              dataRecords={allRecords}
-              showAllStudies={false}
-            ></StudiesTable>
+            <iframe
+              title="References Table"
+              src="https://airtable.com/embed/shraXWPJ9Yu7ybowM?backgroundColor=cyan&viewControls=on"
+              frameBorder="0"
+              width="100%"
+              height="533"
+              style={iframeStyle}
```

Comment:
we should use css classes here instead of inline style, esp if we might want to use this type of styling for other iframes